### PR TITLE
[READY] Wand of nothing locations fixes

### PR DESCRIPTION
**Pull request by itseasytosee**
Finishes #49240 which wasn't quite done

### CODE REVIEW

**Comment by itseasytosee** [comment]
I was working on the fix when it got merged

**Comment by trollbreeder** [comment]
speedmerge now

File: code/modules/projectiles/guns/magic/wand.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/magic/wand
 	name = "wand of nothing"
-	desc = "It's not just a stick, it's a MAGIC stick!"
+	desc = "It's not just a stick, it's a MAGIC stick! You shouldent have this."
```
Comment (SpaceManiac) [inline code/modules/projectiles/guns/magic/wand.dm]:
```suggestion
	desc = "It's not just a stick, it's a MAGIC stick! You shouldn't have this."
```

**Comment by ATH1909** [comment]
Okay, now this is epic.

File: code/modules/projectiles/guns/magic/wand.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/magic/wand
 	name = "wand of nothing"
-	desc = "It's not just a stick, it's a MAGIC stick!"
+	desc = "It's not just a stick, it's a MAGIC stick! You shouldn't have this."
```
Comment (Cyberboss) [inline code/modules/projectiles/guns/magic/wand.dm]: This is also the wand of nothing's description sooo do something about that.

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/projectiles/guns/magic/wand.dm (hunk 2 above)
Comment (itseasytosee) [reply code/modules/projectiles/guns/magic/wand.dm]: fixed

**Comment by itseasytosee** [comment]
All good to go

**Comment by cacogen** [comment]
I think you've missed some

```
  tgstation\code\game\objects\items\storage\belt.dm (8 hits)
	Line 494: 		/obj/item/gun/magic/wand
  tgstation\code\modules\vending\autodrobe.dm (1 hit)
	Line 129: 					  /obj/item/gun/magic/wand = 2,
  tgstation\_maps\map_files\generic\CentCom.dmm (1 hit)
	Line 13806: /obj/item/gun/magic/wand{
  tgstation\_maps\RandomZLevels\SnowCabin.dmm (1 hit)
	Line 4114: /obj/item/gun/magic/wand{
```

**Comment by itseasytosee** [comment]
those a snowflake wands of engine ignition

**Comment by cacogen** [comment]
I see, they're mapedited to be a wand of engine ignition with no uses and a staff that does nothing. They should still use the new wand of nothing path though.

The one in the theatre autodrobe's contraband list definitely needs changing.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**